realm-transformer/src/main/java/io/realm/transformer/RealmAnalytics.java

```diff
@@ -68,6 +68,7 @@
             + "      \"Anonymized MAC Address\": \"%USER_ID%\",\n"
             + "      \"Anonymized Bundle ID\": \"%APP_ID%\",\n"
             + "      \"Binding\": \"java\",\n"
+            + "      \"Uses Kotlin\": \"%USES_KOTLIN%\",\n"
```

Comment:
@timanglade Yeah, I thought about that before starting to code. The problem is that it's very possible (and common) to have hybrid projects, that contain both Java and Kotlin code, so the singular `Language` is a bit of a problem here. That's why I opted for the boolean flag. Checking whether a project if purely a Kotlin one would require the traversal of the whole project tree, which can be time consuming.